doc: release: doc stepper API as major enhancement (#81419)
zephyrproject-rtos/zephyr: doc/releases/release-notes-4.0.rst

  :dtcompatible:`nordic,nrf-comp`, :dtcompatible:`nordic,nrf-lpcomp` and
  :dtcompatible:`nxp,kinetis-acmp` are supported.

> * The introduction of the :ref:`stepper<stepper_api>` device driver subsystem for stepper motors,
>   complete with shell support. It allows initial configuration through devicetree and runtime
>   control as well as configuration through generic as well as vendor specific apis. Initially
>   implemented drivers include a simple :dtcompatible:`zephyr,gpio-stepper` and a complex sensorless
>   stall-detection capable with integrated ramp-controller :dtcompatible:`adi,tmc5041`.

Reviewer: ```suggestion
* The introduction of the :ref:`stepper<stepper_api>` device driver subsystem for stepper motors,
  complete with shell support. Initially implemented drivers include a simple
  :dtcompatible:`zephyr,gpio-stepper` and a complex sensor-less stall-detection capable with
  integrated ramp-controller :dtcompatible:`adi,tmc5041`.
```

proposing to drop the second sentence for the sake of keeping the "major enhancement" section to the point -- people can always learn more by reading the rest of the full release notes.
